test/integration/scenarios/04.workflow/manifest/campaign.yaml

```diff
@@ -32,7 +32,7 @@ spec:
         password: ""
       inputs:
         objectType: catalogs
-        namesOnly: true
+        namesOnly: "${{$trigger(namesOnly,false)}}"
```

Comment:
How about the samples in doc folder, do we need to modify them as well?

Reply:
I think we can keep it as is. `trigger` function is optional.

Reply:
But in current behavior, how about if activation has an input, e.g. nameOnly and campaign doesn't have it. How to refer this property from activation?

Reply:
In the current behavior, we should use `${{$input(nameOnly)}}` and ensure the inputs in campaign stage don't overwrite the activation inputs.

Reply:
Thanks for the detailed discussion, @lirenjie95 , @RemindD . 
By introducing $trigger, this means we always rely on campaign users to specify whether the campaign takes the input from activation or not. This is different from current experience. Campaign users can write counter stage provider without any inputs. And let activation user to provide the inputs in activations. 
<img width="727" alt="image" src="https://github.com/user-attachments/assets/be192983-38b4-464d-9f25-baf717dc3669">
<img width="437" alt="image" src="https://github.com/user-attachments/assets/d7841092-8edd-4f88-9182-3f6c702a8028">

We have to change the campaign spec like this,
<img width="835" alt="image" src="https://github.com/user-attachments/assets/9208e351-8eeb-4ec0-bb40-fa05ac755b3e">

This will bring extra cost for campaign users to maintain the campaign spec. Will this affect users' experience? Also cc @Haishi2016 .

From my point of view, I think activation.input should be per stage and always has higher priority than stage.input in campaign. So I think we even don't need to provider $trigger.
If we think we need to give the freedom to campaign users to choose whether they want to respect activation.input or not. We can enforce campaign users to write $trigger in every campaign. But I feel this is a little duplicated. Maybe we can have default behavior, like when one field shows up in activation.input while misses in stage.input, we can always take activation.input as input.

Reply:
Thanks for sharing the details, @Haishi2016 . @lirenjie95 / @RemindD  Since our goal is to distinguish activation.input and stage.input. I think to maintain the clear boundary and avoid providing shortcut to take the activation.input automatically is better. Then I am good with existing implementation. 
Our expected behavior is campaign writer needs to always explicitly specify whether want to use activation input by $trigger.

Reply:
doc is ready (in `campaign.md`)